style/app.scss

```diff
@@ -218,9 +218,19 @@ body
   padding-bottom : 3px;
 }
 
-#nav-export
+#export
 {
-  cursor : pointer;
+  cursor: pointer;
+  position: absolute;
```

Comment:
Similar to my comment on the other PR, you should try avoiding using absolute positioning.

This is a bit trickier to implement, especially after the nav bar changes in #1219 (which reminds, me you should update from master). A modern CSS approach is to use [**Flexbox**](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) on the nav bar (set `display: flex` and the `flex-direction: row`), and then have the three elements be the logo, the link `ul`, and the export button. After you get the general positioning right, you can use `text-align: right` on the button to make sure it's flush to the right.